README.md

```diff
@@ -11,24 +8,23 @@ A Neural Net providing a subgrid parameterization of atmospheric convection in a
 The work is described in a [GRL paper](https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2020GL091363) with [accompanying code available](https://github.com/yaniyuval/Neural_nework_parameterization/tree/v.1.0.3). The repository contains the neural net and its implementation into a simple system for atmospheric modelling, [SAM](http://rossby.msrc.sunysb.edu/~marat/SAM.html).
 
 The aims of this repository are to:
-1. develop a standalone fortran module based on this neural net that can be used elsewhere,
+1. provide a standalone fortran module of the convection parameterisation based on this neural net that can be used elsewhere,
 2. deploy the module in another atmospheric model, and
-3. evaluate its performance.
+3. evaluate its performance (and provide instructions/code to do this).
```

Comment:
I guess this is now done in https://github.com/m2lines/CAM-ML.
Initially we were advised that all the work would be done here, but later it was suggested that we use a fork of CAM to implement it. So perhaps this needs changing now.
Though evaluation results might be reported on this repo once we finalise them.

Reply:
Do you have any suggestion on how to change the text here? Or leave it like this?

Reply:
Not really...
I think leave for now perhaps.
I think the CAM-ML repo is linked from elsewhere in the README.